build.gradle

```diff
@@ -666,66 +666,7 @@ task combineUnpickDefinitions(type : FileOutput) {
 	}
 }
 
-task remapUnpickDefinitionsIntermediary(type : FileOutputInput, dependsOn: [combineUnpickDefinitions, buildYarnTiny]) {
```

Comment:
The output in the jar is now going to be wrong, this needs to be changed to remap to named to be included in the exported jars.

Reply:
What? Unpick is intermediary and applied to intermediary (unpicked intermediary). named jar has always been unpicked and derived from the unpicked intermediary.

I honestly don't know what you are talking about, as I am looking at the cfr decompiled result and it has no problems for remapping (except those ones where I explicitly override the desc)

Reply:
The published jar files that loom use need to contain named unpick definitions, this is no longer the case.

Reply:
Done.